geopandas/tools/tests/test_sjoin_old.py

```diff
@@ -227,7 +225,6 @@ def test_sjoin_values(self):
         self.assertEquals(df.shape, (12,8))
 
     @unittest.skipIf(str(pd.__version__) < LooseVersion('0.19'), pandas_0_18_problem)
-    @pytest.mark.xfail
```

Comment:
this one is still failing (so the only sjoin test that should fail), but with an interesting error somewhere in the pandas munging of the frames. So one I have to look at when improving the pandas interactions, therefore removed it for now